From bee2402764459feb49c88d4891e77c98eedd9ffd Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 16 May 2019 09:32:29 +0800
Subject: [PATCH 2/3] mtd: fsl-quadspi: add fsl-quadspi enabling for i.mx8mq

Allow fsl-quadspi building for i.mx8mq then we don't need
use it by enabling ARCH_LAYERSCAPE.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mtd/spi-nor/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/spi-nor/Kconfig b/drivers/mtd/spi-nor/Kconfig
index dbe4c4196d31..6790bb2e4845 100644
--- a/drivers/mtd/spi-nor/Kconfig
+++ b/drivers/mtd/spi-nor/Kconfig
@@ -61,7 +61,7 @@ config SPI_CADENCE_QUADSPI
 
 config SPI_FSL_QUADSPI
 	tristate "Freescale Quad SPI controller"
-	depends on ARCH_MXC || SOC_LS1021A || ARCH_LAYERSCAPE || COMPILE_TEST
+	depends on ARCH_MXC || SOC_LS1021A || ARCH_LAYERSCAPE || ARCH_FSL_IMX8MQ || COMPILE_TEST
 	depends on HAS_IOMEM
 	help
 	  This enables support for the Quad SPI controller in master mode.
-- 
2.17.1

